internal/meta_client.go

```diff
@@ -51,120 +51,158 @@ type MetaClientMock struct {
 	UsersFn                  func() []meta.UserInfo
 }
 
+// Close a MetaClientMock
 func (c *MetaClientMock) Close() error {
 	return c.CloseFn()
 }
 
+// CreateContinuousQuery creates a mock of MetaClient.CreateContinuousQuery
```

Comment:
These methods don't really "create mocks". What they do is call the `xxxFn` methods on the Mock type. I would change all of them to say the following:

```
CreateContinuousQuery calls the CreateContinuousQueryFn method on the mock MetaClient.
```

Obviously for each method you would switch out `CreateContinuousQuery` for the method's name.